packages/legend-application-query/src/components/QueryEditor.tsx

```diff
@@ -408,7 +487,44 @@ const QueryEditorHeaderContent = observer(
         { persist: true },
       );
     };
+
+    const renameQuery = (): void => {
```

Comment:
could you pls explain where we change the query name?

Reply:
Collected from query editor store when making new query export state, within `this.queryName =
      editorStore.titleOfQuery ?? config.defaultName ?? 'New Query'` since we have moved title of query to be accessible with editorStore now :D